tests/app/notifications/rest/test_callbacks.py

```diff
@@ -297,36 +271,19 @@ def test_firetext_callback_should_update_multiple_notification_status_sent(
     notification3 = create_sample_notification(
         notify_db, notify_db_session, status='sending', sent_at=datetime.utcnow()
     )
+    data = 'mobile=441234123123&status=0&time=2016-03-10 14:17:00&reference={}'.format(
+        notification1.id)
 
-    client.post(
-        path='/notifications/sms/firetext',
-        data='mobile=441234123123&status=0&time=2016-03-10 14:17:00&reference={}'.format(
-            notification1.id
-        ),
-        headers=[('Content-Type', 'application/x-www-form-urlencoded')])
-
-    client.post(
-        path='/notifications/sms/firetext',
-        data='mobile=441234123123&status=0&time=2016-03-10 14:17:00&reference={}'.format(
-            notification2.id
-        ),
-        headers=[('Content-Type', 'application/x-www-form-urlencoded')])
-
-    client.post(
-        path='/notifications/sms/firetext',
-        data='mobile=441234123123&status=0&time=2016-03-10 14:17:00&reference={}'.format(
-            notification3.id
-        ),
-        headers=[('Content-Type', 'application/x-www-form-urlencoded')])
+    firetext_post(client, data)
+    firetext_post(client, data)
+    firetext_post(client, data)
```

Comment:
this test isn't the same - before it was sending for three separate notifications one after the other, and now it's sending for the first notification three times.

this test has also never had any asserts 😱 

Can you make it send for the three notifications again, and also, assert something relevant?

Or alternatively we can remove it, I'm not sure how much value this test is providing.